src/Http/Http.Extensions/src/RequestHeaders.cs

```diff
@@ -279,11 +279,26 @@ public Uri Referer
             }
         }
 
+        /// <summary>
+        /// Gets the value of header with <paramref name="name"/>.
+        /// </summary>
+        /// <typeparam name="T">The type of the header.
+        /// The given type must have a static TryParse method</typeparam>
+        /// <param name="name">The name of the header to retrieve.</param>
+        /// <returns>The value of the header</returns>
```

Comment:
As a follow-up, we should either make another PR or add to this PR to add xml documentation for the rest of these properties. 

Also, isn't there an analyzer that we can enable to check for all public apis that don't have xml docs?

Reply:
One small nit:

```suggestion
        /// <returns>The value of the header.</returns>
```

> Also, isn't there an analyzer that we can enable to check for all public apis that don't have xml docs?

It's even better than that :). If we turn the right settings on we get a build warning (or error if we promote it to that). Some projects are using that now. We need to get "clean" before we can turn that on more broadly though.

Reply:
@jkotalik and @anurse thanks for the comments! They should be resolved in the latest commit.

Regarding the other properties in these classes, I'm willing to document them as well, but do add those docs to this PR or do I open a new PR?